capitoli/Nono_Passo_Liste.rst

```diff
@@ -253,7 +254,7 @@ Una sezione di lista è l'analogo di una sezione di stringa e ha la stessa logi
 Se *amici* é la lista *[“Gianni”,”Luigi”,”Carlo”]*
 ::
 
-	print amici [0:3])
+	print (amici [0:3])
 
 produce: *"Gianni" "Luigi" "Carlo"*
```

Comment:
occhio anche nel caso di 
```
allievo = allievi_4E [2]
allievo [0]
Rossi
```
perchè manca la print prima di allievo[0]

Reply:
Problema di nuovo in
```
while i < len(allievi):
        print (allievi[i])
i=i+ 1
```
Perchè i=i+1 deve essere indentato sennò non incremento nell'iterazione